Source: oahong@gmail.com
Upstream: No
Reasons: Fix as-needed issue.
--- gcin-1.4.5.orig/Makefile	2009-03-29 21:32:52.000000000 +0800
+++ gcin-1.4.5/Makefile	2009-05-02 17:04:27.075517936 +0800
@@ -135,7 +135,7 @@
 	$(CC) -o $@ $(OBJS_phoa2d) $(LDFLAGS)
 
 phod2a: $(OBJS_phod2a)
-	$(CC) -o $@ $(OBJS_phod2a) $(LDFLAGS)
+	$(CC) -lX11 -o $@ $(OBJS_phod2a) $(LDFLAGS)
 
 tsa2d:  $(OBJS_tsa2d)
 	$(CC) -o $@ $(OBJS_tsa2d) $(LDFLAGS)
